Jenkinsfile_k8s

```diff
@@ -6,9 +6,9 @@ podTemplate(
             command: 'cat',
             ttyEnabled: true,
             resourceRequestCpu: '200m',
-            resourceLimitCpu: '200m',
+            resourceLimitCpu: '400m',
             resourceRequestMemory: '256Mi',
-            resourceLimitMemory: '256Mi',
+            resourceLimitMemory: '1024Mi',
```

Comment:
- You might want to define the same values for limits and requests (for both CPU and mem)
- Maven is a compiler: don't hesitate to allocate resources "in mass": 2 vCPUs  + 2 Gb memory are quite common for building an app.

Reply:
There were "debug" values, it worked with 1024, I'm trying with 512 only, will see if it passes

Reply:
512 wasn't enough, I've put the values you suggested.